agent/mission_solver/solver.py

```
@@ -0,0 +1,137 @@
+    # 设置变量为整数类型
+    for i in range(fn):
+        h.changeColIntegrality(i, highspy.HighsVarType.kInteger)
+
+    # 添加约束: Σ(A[j][i] * x_i) >= b[j]
+    for j in range(m):
+        indices = [i for i in range(fn) if filtered_A[j][i] > 0]
+        values = [float(filtered_A[j][i]) for i in indices]
+        if indices:
+            h.addRow(float(valid_missions[j].count), inf, len(indices), indices, values)
+        else:
+            logger.warning(f"任务 '{valid_missions[j].description}' 无匹配副本")
```

Ruff (0.15.11)

[warning] Logging statement uses f-string

(G004)